test/CliTest.js

```diff
@@ -12,6 +12,8 @@ const CLI = require('../src/CLI.js'),
   fs = require('fs'),
   nock = require('nock');
 
+require('events').EventEmitter.defaultMaxListeners = Infinity;
```

Comment:
This line removes event emitter max listeners limit. This line was removed and the generated warning will be investigated in #27